lib/order.rb

```diff
@@ -0,0 +1,36 @@
+module Order
+=begin
```

Comment:
Style/BlockComments: Do not use block comments. (https://github.com/bbatsov/ruby-style-guide#no-block-comments)